portal/app/[locale]/tunnel/_components/evmWithdrawalWaitTime.tsx

```diff
@@ -0,0 +1,106 @@
+import { useNetworkType } from 'hooks/useNetworkType'
+import { useTranslations } from 'next-intl'
+import Skeleton from 'react-loading-skeleton'
+import { MessageStatus, ToEvmWithdrawOperation } from 'types/tunnel'
+import { secondsToHours, secondsToMinutes } from 'utils/time'
+
+import {
+  useEvmWithdrawTimeToFinalize,
+  useEvmWithdrawTimeToProve,
+} from '../_hooks/useWaitTime'
+
+type Props = {
+  withdrawal: ToEvmWithdrawOperation
+}
+
+const ExpectedWithdrawalWaitTimeSecondsTestnet = 20 * 60 // 20 minutes
+const ExpectedWithdrawalWaitTimeSecondsMainnet = 40 * 60 // 40 minutes
+const ExpectedProofWaitTimeSecondsTestnet = 3 * 60 * 60 // 3 hours
+const ExpectedProofWaitTimeSecondsMainnet = 24 * 60 * 60 // 24 hours
+
+const renderTime = function (
```

Comment:
It looks like this function could better be a component that renders the `<span>` with the text inside. It could also call `useTranslations` instead of receiving the `tCommon` as a parameter. In addition, to be reused, it could receive the type of time: withdrawal or proof.

I think it is abstracting some logic but the function/component could do better in that regard.